### PR TITLE
Heroku seeds

### PR DESCRIPTION
**Pull request by NessEFC**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -49,7 +49,7 @@ group :development, :test do
   gem 'database_cleaner'
   gem 'rspec-rails'
   gem 'awesome_print', require:'ap'
-  gem "nyan-cat-formatter"
+  gem 'nyan-cat-formatter'
```
Comment (NessEFC) [inline Gemfile]: Way to be consistent with your quotes.

File: spec/features/user_can_view_an_order_show_page_spec.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.feature "As a logged-in user," do
+  scenario "they can view a single order's show page" do
+
+    user = create(:user)
+    order = user.orders.create(status: "completed")
+    escape = user.orders.first.escapes.create(name: "Gentle Dawn", description: "The gentlest of dawns.", price: 500.00, city: "San Diago", image: "url")
+    quantity = 1
```
Comment (caward12) [inline spec/features/user_can_view_an_order_show_page_spec.rb]: we will need to fix this when we have full checkout functionality

File: spec/features/user_can_view_an_order_show_page_spec.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    total = "$500.00"
```
Comment (caward12) [inline spec/features/user_can_view_an_order_show_page_spec.rb]: again, fix when full checkout functionality

File: app/views/orders/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<h1>Order #<%= @order.id %></h1>
+
+
+Order Date: <%= @order.format_created_at %><br>
+Order Status: <%= @order.status %>
+<% if @order.status == "cancelled" || @order.status == "completed" %>
```
Comment (lukyans) [inline app/views/orders/show.html.erb]: create a method status_with_date in models to refactor it

**Pull request merged**